RELEASE_NOTES.rst

```diff
@@ -4,28 +4,28 @@ coala 0.10 - PolarBear
 ::
 
```

Comment:
You need to make changes in the unittests as well, that's why the CI tests are failing.